templates/admin/failedJobDetails.tpl

```diff
@@ -14,26 +14,28 @@
 		{translate key=$pageTitle}
 	</h1>
 	<div class="app__contentPanel app__contentPanel__overflow">
-		<pkp-table
-			:columns="columns"
-			:rows="rows"
-			:label="label"
-		>
-			<template #default="{ row, rowIndex }">
-				<table-cell
-					v-for="(column, columnIndex) in columns"
-					:key="column.name"
-					:column="column"
-					:row="row"
-					:tabindex="!rowIndex && !columnIndex ? 0 : -1"
-				>
-					<template #default v-if="column.name === 'value' && isValidJson(row.value)">
-						<span>
-							<pre>{{ row.value }}</pre>
-						</span>
-					</template>
-				</table-cell>
-			</template>
+		<div class="bg-default p-4">
+			<caption v-if="label">
+				<div class="pkpTable__label">
```

Comment:
I believe that caption tag is intended to use inside table tag to be correctly associated with that table?

Our new table don't really have capability to define heading/description yet. Reason is really that it was initially created for new submission listing which does not really title/description embeded with table.

<img width="1000" alt="Screenshot 2024-08-07 at 13 21 24" src="https://github.com/user-attachments/assets/8f18c963-4a6f-49da-a4a1-b206bee9d390">

But actually most other tables have quite simple pattern of having heading and description. So I think it make sense to extend the new table component to support that pretty much the same as we had in previous Table.

As result it would be possible to either provide label/description, or also possible to provide the aria variants if we just need to connect with some custom heading etc - aria-labelledBy and aria-describedBy like we will want to do in submission listing for example.

For label/description we can follow some design examples from submission listing:


<img width="300" alt="Screenshot 2024-08-07 at 13 25 50" src="https://github.com/user-attachments/assets/81d55a1a-df62-43d7-ad03-729b99a16ec2">

Reply:
Update here, I just added a `class="!max-w-[8rem] truncate"` directly on my use of TableColumn. It's not common to have long column names in the tables I migrated so I think this would be enough.